tests/integration/test_crunch_cube.py

```diff
@@ -1360,23 +1365,29 @@ def test_pets_x_pets_array_by_col(self):
     def test_pets_x_pets_array_by_row(self):
         cube = CrunchCube(FIXT_PETS_X_PETS_ARRAY)
         expected = np.array([
-            [0.44444444, 0.41176471],
-            [0., 1.],
-            [0.5, 0.47368421],
+            [0.55555556, 0.19444444],
+            [0., 0.55555556],
+            [0.44444444, 0.25],
         ])
-        actual = cube.proportions(axis=2)[0]
-        # Since cube is 3D, col dim is 1 (instead of 0)
+        actual = cube.proportions(axis=1)[0]
         np.testing.assert_almost_equal(actual, expected)
 
     def test_pets_x_pets_array_by_cell(self):
         cube = CrunchCube(FIXT_PETS_X_PETS_ARRAY)
         expected = np.array([
-            [0.44444444, 0.41176471],
-            [0., 1.],
-            [0.5, 0.47368421],
+            [[0.08733624, 0.06113537],
+             [0., 0.17467249],
+             [0.069869, 0.07860262]],
+
+            [[0., 0.14847162],
+             [0.06550218, 0.06113537],
+             [0.05676856, 0.069869]],
+
+            [[0.09606987, 0.069869],
+             [0.08733624, 0.07860262],
+             [0., 0.16593886]]
```

Comment:
👍 these match R's output (the output is a bit hard to read, so I'm attaching R output just to confirm that these get collated into the correct dimensions)

```
> prop.table(pets_array)
, , pets_array = not selected

        pets_array
pets            cat        dog     wombat
  dog    0.08733624 0.00000000 0.06986900
  cat    0.00000000 0.06550218 0.05676856
  wombat 0.09606987 0.08733624 0.00000000

, , pets_array = selected

        pets_array
pets            cat        dog     wombat
  dog    0.06113537 0.17467249 0.07860262
  cat    0.14847162 0.06113537 0.06986900
  wombat 0.06986900 0.07860262 0.16593886
```